features/cxf/src/main/feature/feature.xml

```diff
@@ -233,7 +264,7 @@
         <!-- Can be used instead cxf-http for a smaller footprint -->
         <feature version="${cxf.version}">cxf-core</feature>
         <bundle start-level="40">
-            mvn:${cxf.servlet-api.group}/${cxf.servlet-api.artifact}/${cxf.servlet-api.version}
+            mvn:javax.servlet/javax.servlet-api/${cxf.javax.servlet-api.version}
```

Comment:
How is it that we got to move off the CXF API and onto the javax version of the API?